DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule/Mvc/Controllers/PreSearchFiltersController.cs

```diff
@@ -102,12 +102,15 @@ public class PreSearchFiltersController : BaseDfcController
         [DisplayName("Group By")]
         public string GroupFieldsBy { get; set; } = "Skills";
 
-        [DisplayName("Should show number of profile matched banner")]
+        [DisplayName("Number of profle matching on each page message")]
         public string NumberOfMatchesMessage { get; set; } = "We have found {0} career matches based on your selection.";
 
         [DisplayName("Select Message")]
         public string SelectMessage { get; set; } = @"<div class=""govuk-hint"" id=""qualifications-hint"">Select all that apply.</div>";
 
+        [DisplayName("Use Page Profile Count")]
+        public bool UsePageProfileCount { get; set; } = true;
+
```

Comment:
Does this now affect current unit/acceptance tests or do we need to add some to prove that wehn this is false we con't make a call to GetNumberOfMatches? Happy for this to be done later if this is urgent for perf testing in pre-prod.

Reply:
Should be done now